src/models/data.py

```diff
@@ -9,9 +10,15 @@
 from typing import cast, Dict, Optional, Union, List, Tuple
 
 
+@dataclass
+class TrainData:
+    historical: Union[np.ndarray, torch.Tensor]
+    pred_months: Union[np.ndarray, torch.Tensor]
+
```

Comment:
is the idea here to also have another attribute `additional` or `current_month` perhaps:

I have a line [here](https://github.com/esowc/ml_drought/pull/49/files?file-filters%5B%5D=.py#diff-b07d4def3bfdc11dc410d8848b939e1aR15)
```
current = Union[np.ndarray, torch.Tensor, None]
```

Reply:
yep! This can be extended as necessary